src/lib/sidebar/EntireScheme.svelte

```diff
@@ -1,24 +1,30 @@
 <script lang="ts">
-  import { onMount } from "svelte";
+  import { onMount, onDestroy } from "svelte";
   import length from "@turf/length";
   import {
     gjScheme,
     formOpen,
     mapHover,
     sidebarHover,
     openFromSidebar,
+    currentMode,
   } from "../../stores";
   import type { Scheme, Schema } from "../../types";
+  import ConfirmationModal from "../common/ConfirmationModal.svelte";
+  import type { Mode } from "../../types";
 
   export let authorityName: string;
   export let schema: Schema;
+  let mode: Mode = "edit-attribute";
```

Comment:
We can also use `$currentMode` here, not need the local var that's synced to it, and not need `unsubscribe`